src/util/ecdsa.rs

```diff
@@ -16,9 +16,11 @@
 //! ECDSA keys used in Bitcoin that can be roundtrip (de)serialized.
 //!
 
-use std::fmt::{self, Write};
-use std::{io, ops};
-use std::str::FromStr;
+use core::fmt;
+use core::fmt::Write as _fmtWrite;
+use core::ops;
```

Comment:
That otoh could be written in less lines :laughing:

Reply:
done